corehq/apps/reports/standard/tableau.py

```diff
@@ -63,34 +64,47 @@ def page_context(self):
 
     def get(self, request, *args, **kwargs):
         if self.visualization.server.server_type == 'server':
-            return self.tableau_server_response()
+            return render(self.request, self.template_name, self.get_context_data())
         return super().get(request, *args, **kwargs)
 
-    def tableau_server_response(self):
-        from requests_toolbelt.adapters import host_header_ssl  # avoid top-level import that breaks docs build
-        context = self.get_context_data()
-        tabserver_url = 'https://{}/trusted/'.format(self.visualization.server.server_name)
-        post_arguments = {'username': self.visualization.server.domain_username}
-        if self.visualization.server.target_site != 'Default':
-            post_arguments.update({'target_site': self.visualization.server.target_site})
-
-        if self.visualization.server.validate_hostname != '':
-            request = requests.Session()
-            request.mount('https://', host_header_ssl.HostHeaderSSLAdapter())
-            tabserver_response = request.post(tabserver_url,
-                                              post_arguments,
-                                              headers={'Host': self.visualization.server.validate_hostname})
-        else:
-            tabserver_response = requests.post(tabserver_url, post_arguments)
-
-        if tabserver_response.status_code == 200:
-            if tabserver_response.content != b'-1':
-                context.update({'ticket': tabserver_response.content.decode('utf-8')})
-                return render(self.request, self.template_name, context)
-            else:
-                context.update({"failure_message": _("Tableau trusted authentication failed")})
-                return render(self.request, 'reports/tableau_server_request_failed.html', context)
+
+@require_POST
```

Comment:
This also might as well require tableau permissions and domain login.

Reply:
going to that url without permission currently gives a 403 via the dispatch